ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -31,6 +31,7 @@
   private static final String PROP_ELEVATION = "elevation";
   private static final String PROP_Z_INDEX = "zIndex";
   private static final String PROP_RENDER_TO_HARDWARE_TEXTURE = "renderToHardwareTextureAndroid";
+  private static final String PROP_ACCESSIBLE = "accessible";
```

Comment:
What is this for?  It seems like its functionality could be handled with the existing IMPORTANT_FOR_ACCESSIBILITY prop, and it doesn't see, to be used in the rest of this PR.